src/Nethermind/Nethermind.AccountAbstraction/Network/UserOperationsMessageSerializer.cs

```diff
@@ -15,7 +15,7 @@ public class UserOperationsMessageSerializer : IZeroInnerMessageSerializer<UserO
         public void Serialize(IByteBuffer byteBuffer, UserOperationsMessage message)
         {
             int length = GetLength(message, out int contentLength);
-            byteBuffer.EnsureWritable(length, true);
+            byteBuffer.EnsureWritable(length);
```

Comment:
What is the difference?

Reply:
The `true` parameter forces the allocation regardless of whether the buffer is over settings; without it the call throws if over settings